wiki/Tournaments/RMoT_Invitational/5/en.md

```diff
@@ -0,0 +1,167 @@
+# Rapid Monthly osu! Tournament Invitational 5
+
+The Rapid Monthly osu! Tournament Invitational \#5 is a tournament organized and run by the Rapid Tournaments team.
+It is an exclusive tournament where only players who have reached semi-finals in one of the last four RMoT's get invited.
+
+The tournament brackets can be found on the [Challonge page](https://challonge.com/rmotinvitational5). Most of the information here complies with the [forum post](/community/forums/topics/741126).
+
+![France][flag_FR] [Prosdokies](/users/3852436 "Prosdokies") won the fifth Invitational, beating ![Russian Federation][flag_RU] [Well Halen](/users/8660244 "Well Halen") in the Grand Finals
+
+## Schedule
+
+This RMoT Invitational was held over three weekends.
+
+| Event                               | Date       |
+| ----------------------------------: | :--------- |
+| Round of 16                         | 2018-05-12 |
+| Quarterfinals and Losers Round 1    | 2018-05-13 |
+| Semifinals and Losers Round 2       | 2018-05-19 |
+| Finals and Losers Round 3 and 4     | 2018-05-20 |
+| Losers Semifinals and Losers Finals | 2018-05-26 |
+| Grand Finals                        | 2018-05-27 |
+
+## Staff
+
+| Role              | Member                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| ----------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
lol, this header is so long I wouldn't bother with making it match the size of the content. just `---` is fine here

I don't think anyone uses a small enough font for this to have any purpose here